ruby/build_jruby_release.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# This file should be executed with jruby
+
+set -ex
+
+WORKING_DIR=$(dirname $(readlink $BASH_SOURCE))
```

Comment:
If you use inline_sh_binary instead of sh_binary, you can take advantage of Bazel make variables.  Right now, this is highly non-hermetic, and runs over the non-sandboxed source tree